src/Hyperwallet.js

```diff
@@ -2016,7 +2016,11 @@ export default class Hyperwallet {
      * @param {Object} inputFilters - The query parameters in the URI
      * @param {Object} listFilters - Defined list of filters for a business object
      */
-    isValidFilter(inputFilters, listFilters) {
-        return Object.keys(inputFilters).every(elem => listFilters.includes(elem));
+    isValidFilter(options, listFilters) {
+        const optionKeys = Object.keys(options);
+        if (optionKeys.length === 0) {
+            return true;
+        }
+        return listFilters.every(elem => optionKeys.includes(elem));
```

Comment:
Think the logic here is incorrect. Please double-check.

Reply:
The logic:
 Check if there are no query options in the request body:
     - if the options dont have any parameter, return that its a valid query body
     - If the options dont contain any of the mandatory fields, then return that its an invalid query body. 
    
Please let me know if the code change dont satisfy the logic mentioned above.

Reply:
Reversed the logic as discussed and tested with data. Please review and approve.